### PR TITLE
Add READNE.md and executor tests for trino

### PR DESCRIPTION
**Pull request by ws00428637**
### What is the purpose of the change

Add some guide for Trino EngineConn plugin.

### Related issues/PRs

Related issues: #2687
Related pr:#2639

### Brief change log

- Add Trino executor test class;
- Add Trino README.md;
- Fix some configurations for Trino.

### Checklist

- [x] I have read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/main/CONTRIBUTING.md#pull-requests).
- [x] I have explained the need for this PR and the problem it solves
- [x] I have explained the changes or the new features added to this PR
- [x] I have added tests corresponding to this change
- [x] I have updated the documentation to reflect this change
- [x] I have verified that this change is backward compatible (If not, please discuss on the [Linkis mailing list](https://linkis.apache.org/community/how-to-subscribe) first)
- [x] **If this is a code change**: I have written unit tests to fully verify the new behavior.

### CODE REVIEW

**Review by peacewong — DISMISSED** [review]
LGTM.

File: linkis-engineconn-plugins/trino/README.md [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+```
+${linkis_code_dir}/linkis-engineconn-plugins/trino/
+mvn clean install
+```
+
+### 2.2 物料的部署和加载
+
+将 2.1 步编译出来的引擎包,位于
+```bash
+${linkis_code_dir}/linkis-engineconn-plugins/jdbc/target/out/trino
```
Comment (casionone) [inline linkis-engineconn-plugins/trino/README.md:30]: ${linkis_code_dir}/linkis-engineconn-plugins/trino/target/out/trino

File: linkis-engineconn-plugins/trino/README.md [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+| linkis.trino.ssl.keystore.password     | null                  |   否   | Trino服务器SSL keystore密码 |
+| linkis.trino.ssl.truststore            | null                  |   否   | Trino服务器SSL truststore路径 |
+| linkis.trino.ssl.truststore.type       | null                  |   否   | Trino服务器SSL truststore类型 |
+| linkis.trino.ssl.truststore.password   | null                  |   否   | Trino服务器SSL truststore密码 |
+
+## 3. Trino
+### 3.1 准备操作
+您需要配置Trino的连接信息，包括连接地址信息或用户名密码(如果启用)等信息。
+
+![ElasticSearch](https://user-images.githubusercontent.com/22620332/182787195-8051bf25-1e1e-47e5-ad88-4896278857f2.png)
```
Comment (casionone) [inline linkis-engineconn-plugins/trino/README.md:127]: should adjust to  actual trino engine png  
not elasticSearch

File: linkis-engineconn-plugins/trino/README.md [hunk 3]
```diff
@@ -0,0 +1,151 @@
+---
+title: Trino 引擎
```
Comment (casionone) [inline linkis-engineconn-plugins/trino/README.md:2]: the doc file  is batter to put at `docs/trino-usage.md`

**Comment by casionone** [comment]
```
2022-10-17T03:14:38.5724692Z java.lang.NullPointerException: session is null
2022-10-17T03:14:38.5727761Z 	at java.util.Objects.requireNonNull(Objects.java:228)
2022-10-17T03:14:38.5731705Z 	at io.trino.client.StatementClientV1.<init>(StatementClientV1.java:96)
2022-10-17T03:14:38.5736995Z 	at io.trino.client.StatementClientFactory.newStatementClient(StatementClientFactory.java:24)
2022-10-17T03:14:38.5743771Z 	at org.apache.linkis.engineplugin.trino.executor.TrinoEngineConnExecutor.executeLine(TrinoEngineConnExecutor.scala:190)
2022-10-17T03:14:38.5750932Z 	at org.apache.linkis.engineplugin.trino.executer.TestTrinoEngineConnExecutor.testExecuteLine(TestTrinoEngineConnExecutor.scala:112)
```

**Review by casionone — APPROVED** [review]

**Pull request merged**